### DISCUSSION TITLE
[Feature] Auto-display local albums

### DISCUSSION THREAD

**Discussion by Lakhtion** [immich-app/immich#14413] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

At the moment, the logic of the android mobile application is that I need to select the local albums that I want to sync, and only then will they be displayed in the Library - On this device section, and by default they are not visible, and are not visible unless synchronization is enabled for them.

I did not find a way to exclude an album from synchronization, but display it in this section.

I understand that I can select an album Recent to show all local albums by default, but they will all sync, which I don't want. If I exclude an album from syncing, it won't be visible.

I would like it to be the other way around: by default, Immich would display all local albums, including those newly created during the use of the smartphone, and the user could choose what to display, what to synchronize and whether to show new albums automatically in the application.
This would be logical in the scenario of using Immich as the only universal mobile gallery with cloud synchronization, similar to how it is implemented in Google Photo. At the moment, due to the above logic of working with local albums, the application cannot completely replace the system gallery and you have to use the system gallery in parallel with Immich.

Version of android app and server: 1.121.0 build.168,

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2266
